src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -496,7 +496,37 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                         }
                     }
 
-                    GetEmitter()->emitIns_R_R_R(insEmbMask, emitSize, targetReg, maskReg, embMaskOp1Reg, opt);
+                    if (intrinEmbMask.codeGenIsTableDriven())
```

Comment:
Will `codeGenIsTableDriven` always hold going forwards for additional intrinsics?

I wonder if it needs to be:

```
                        switch (intrinEmbMask.id)
                        {
                            case NI_Sve_LoadVectorByteNonFaultingZeroExtendToInt16:
                            case NI_Sve_LoadVectorByteNonFaultingZeroExtendToInt32:
                            ....
                            case NI_Sve_LoadVectorUInt32NonFaultingZeroExtendToInt64:
                            case NI_Sve_LoadVectorUInt32NonFaultingZeroExtendToUInt64:
                              GetEmitter()->emitIns_R_R_R_I ....

                            default:
                                GetEmitter()->emitIns_R_R_R .....
                                break;
                        }
```

Maybe for now it's fine.

Reply:
It will hold for additional intrinsics that do not have the "HW_Flag_SpecialCodeGen" flag.